src/components/date.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+
+/**
+ * Interface for the DateStr prop
+ */
+interface DateStr {
+    date: string,
+    locals: string,
+    options: any
```

Comment:
Should this be `options: Record<string, string>`?

Reply:
that actually does not work since some stuff in options are not string